testhelper/certificatetest.go

```diff
@@ -76,15 +76,15 @@ func GetRevokableRSALeafCertificate() RSACertTuple {
 }
 
 // GetRevokableRSAChain returns a chain of certificates that specify a local OCSP server signed using RSA algorithm
-func GetRevokableRSAChain(size int) []RSACertTuple {
+func GetRevokableRSAChain(size int, enabledOCSP, enabledCRL bool) []RSACertTuple {
```

Comment:
This is also considered as breaking change.

Reply:
Updated.